ui/public/locales/en.json

```diff
@@ -2447,6 +2448,8 @@
 "message.action.patch.router": "Please confirm that you want to live patch the router. <br> This operation is equivalent updating the router packages and restarting the Network without cleanup.",
 "message.action.patch.systemvm": "Please confirm that you want to patch the System VM.",
 "message.action.primarystorage.enable.maintenance.mode": "Warning: placing the primary storage into maintenance mode will cause all Instances using volumes from it to be stopped.  Do you want to continue?",
+"message.action.primary.storage.scope.cluster": "Change the primary storage scope from zone to cluster",
```

Comment:
These should be before `message.action.primarystorage.enable.maintenance.mode`

Reply:
Done